packages/zwave-js/src/lib/commandclass/DoorLockCC.ts

```diff
@@ -479,6 +535,43 @@ latch status:       ${status.latchStatus}`;
 		// Remember that the interview is complete
 		if (!hadCriticalTimeout) this.interviewComplete = true;
 	}
+
+	public handleLockOperationNotification(isLocked: boolean): void {
+		const node = this.getNode()!;
+		const endpoint = this.getEndpoint()!;
+		const api = endpoint.commandClasses["Door Lock"].withOptions({
+			priority: MessagePriority.NodeQuery,
+		});
+
+		const valueDB = node.valueDB;
+		valueDB.setValue(
+			getCurrentModeValueId(this.endpointIndex),
+			isLocked ? DoorLockMode.Secured : DoorLockMode.Unsecured,
+		);
+		if (valueDB.getValue(getDoorSupportedValueId(this.endpointIndex))) {
+			valueDB.setValue(
+				getDoorStatusValueId(this.endpointIndex),
+				isLocked ? "closed" : "open",
+			);
+		}
+		if (valueDB.getValue(getBoltSupportedValueId(this.endpointIndex))) {
+			valueDB.setValue(
+				getBoltStatusValueId(this.endpointIndex),
+				isLocked ? "locked" : "unlocked",
+			);
+		}
+		if (valueDB.getValue(getLatchSupportedValueId(this.endpointIndex))) {
+			valueDB.setValue(
+				getLatchStatusValueId(this.endpointIndex),
+				isLocked ? "closed" : "open",
+			);
+		}
```

Comment:
This is the same as the door status.  Locking or unlocking the deadbolt tells us nothing about the status of the door latch.